sdk/keyvault/azkeys/client.go

```diff
@@ -1351,26 +1338,25 @@ func (c *Client) GetKeyRotationPolicy(ctx context.Context, keyName string, optio
 	return getKeyRotationPolicyResponseFromGenerated(resp), nil
 }
 
-// ReleaseKeyOptions contains optional parameters for Client.ReleaseKey
+// ReleaseKeyOptions contains optional parameters for Client.ReleaseKey.
 type ReleaseKeyOptions struct {
-	// The version of the key to release
+	// Version is the version of the key to release
 	Version string
 
-	// The encryption algorithm to use to protected the exported key material
+	// Enc is the encryption algorithm used to protected exported key material.
 	Enc *ExportEncryptionAlg `json:"enc,omitempty"`
```

Comment:
I don't know if this was covered in the API review, but `Enc` is a bit too terse and non-descript IMO.

Reply:
Agreed. Pretty sure we want `Algorithm` here, which is what other languages chose. I'll open a PR.